jujugui/static/gui/src/app/app.js

```diff
@@ -2927,6 +2901,7 @@ YUI.add('juju-gui', function(Y) {
         console.log('switching models is not supported in sandbox');
       }
       console.log('switching model connection');
+      this.env.loading = true;
```

Comment:
The "somewhere else" is here: https://github.com/juju/juju-gui/pull/2656/files#diff-21dd467cb411bdd922556d1af7461864R2919